dbt Cloud RN: Private packages must be cloned (#2619)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/02-Dec-2022/private-packages-clone-git-token.md

---
title: "Private packages must be cloned using access tokens provided by environment variables"
description: "Private GitHub packages must be cloned using access tokens provided by environment variables."
sidebar_label: "Deprecation: Private packages must be cloned using access tokens"
tags: [Dec-23-2022]
---

> The supported way for cloning private GitHub packages is to use the [git token method](/docs/build/packages#git-token-method), where an appropriate access token is passed into the package repository URL with an environment variable. 

Reviewer: ```suggestion
The supported method for cloning private GitHub packages is the [git token method](/docs/build/packages#git-token-method), where an appropriate access token is passed into the package repository URL with an environment variable. 
```

> There is a small number of people that has been able to clone private packages using dbt's native GitHub application without explicitly providing an access token. This functionality is being deprecated as it’s limited in flexibility. 

Reviewer: ```suggestion
A small number of people have been able to clone private packages using dbt's native GitHub application without explicitly providing an access token. This functionality is being deprecated as it’s limited in flexibility. 
```
